src/wp-includes/admin-bar.php

```diff
@@ -897,7 +897,7 @@ function wp_admin_bar_edit_menu( $wp_admin_bar ) {
 					)
 				);
 			}
-		} elseif ( is_a( $current_object, 'WP_User' ) && current_user_can( 'edit_user', $current_object->ID ) ) {
+		} elseif ( ( $current_object instanceof \WP_User ) && current_user_can( 'edit_user', $current_object->ID ) ) {
```

Comment:
As none of these files are namespaced, should this be `WP_User` (without the leading backslash ? (here and in the other places)

Reply:
Makes sense, and goes with the pattern used in other `instanceof` uses we have too. Thank you @jrfnl I will force-push with namespace qualifiers removed.